website/documentation/050-tutorials/content/howto/shoot_istio_dns_certs/_index.md

```diff
@@ -0,0 +1,464 @@
+![access](./access.png)
+For convinience, let us set an alias command with 
+```bash
+alias kgarden="kubectl --kubeconfig ~/.kube/kubeconfig-garden-myproject.yaml"
+``` 
+`kgarden` now gives you all botanical powers and connects you directly with your Gardener.
+
+You should now be able to run `kgarden get shoots`, automatically get an oidc token, and list already running clusters/shoots.
+
+## Prepare your Custom Domain
+I am going to use [Cloud Flare](https://www.cloudflare.com/) as programmatic DNS of my custom domain `mydomain.io`. Please follow detailed instructions from Cloud Flare on how to delegate your domain (the free account does not support delegating subdomains). Alternatively, AWS Route53 (and most others) support [delegating subdomains](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html).
+
+I needed to follow these [instructions](https://github.com/gardener/external-dns-management/blob/master/doc/cloudflare/README.md) and created the following secret:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloudflare-mydomain-io
+type: Opaque
+data:
+  CLOUDFLARE_API_TOKEN: useYOURownDAMITzNDU2Nzg5MDEyMzQ1Njc4OQ==
+```
+Apply this secret into your project with `kgarden create -f cloudflare-mydomain-io.yaml`.
+
+Our [External DNS Manager](https://github.com/gardener/external-dns-management/) also supports Amazon Route53, Google CloudDNS, AliCloud DNS, Azure DNS, or OpenStack Designate. Check it out.
+
+## Prepare Gardener Extensions
+I now need to prepare the Gardener extensions `shoot-dns-service` and `shoot-cert-service` and set the parameters accordingly.
+
+The following snipplet allows Gardener to manage my entire custom domain, whereas with the `include:` attribute I restrict all dynamic entries under the subdomain `gsicdc.mydomain.io`:
+
+```yaml
+  dns:
+    providers:
+      - domains:
+          include:
+            - gsicdc.mydomain.io
+        primary: false
+        secretName: cloudflare-mydomain-io
+        type: cloudflare-dns
+  extensions:
+    - type: shoot-dns-service
+```
+
+The next snipplet allows Gardener to manage certificates automatically from *[Let's Encrypt](https://letsencrypt.org/)* on `mydomain.io` for me:
+```yaml
+  extensions:
+    - type: shoot-cert-service
+      providerConfig:
+        apiVersion: service.cert.extensions.gardener.cloud/v1alpha1
+        issuers:
+          - email: me@mail.com
+            name: mydomain
+            server: 'https://acme-v02.api.letsencrypt.org/directory'
+          - email: me@mail.com
+            name: mydomain-staging
+            server: 'https://acme-staging-v02.api.letsencrypt.org/directory'
+```
+
+{{% notice note %}}
+<p>Adjust the snipplets with your parameters (don't forget your email). And please use the mydomain-staging issuer while you are testing and learning. Otherwise, Let's Encrypt will rate limit your frequent requests and you can wait a week until you can continue.</p>
```

Comment:
Good point to refer to https://letsencrypt.org/docs/rate-limits/ and https://letsencrypt.org/docs/staging-environment/